napalm_junos/junos.py

```diff
@@ -51,6 +51,10 @@ def open(self):
         except ConnectTimeoutError as cte:
             raise ConnectionException(cte.message)
         self.device.timeout = self.timeout
+        if hasattr(self.device, "cu"):
```

Comment:
You are right, you can do this now, like for any other driver.

We can add it in ```close``` too, no problem.

In the issue I have not specified all the reasons, I thought it's pointless: without going into deep details, there are cases on a specific Juniper platform where the ```mgd``` process gets stuck at some point (JTAC case underway). Afterwards, communication via netconf does not exist anymore. Even ```close``` will raise error, so this cannot help at all.